docs/source/tutorials/automated-publication-with-ci.md

```diff
@@ -0,0 +1,160 @@
+with the scopes `deposit:actions` and `deposit:write`.
+
+Copy the newly created token into a new [GitHub Secret](https://docs.github.com/en/actions/security-guides/encrypted-secrets#creating-encrypted-secrets-for-a-repository) called `ZENODO_SANDBOX` in your repository.
+
+## Configure a GitHub Action to automate publication 
+
+The HERMES project provides templates for continous integration systems in a dedicated repository:
+https://github.com/hermes-hmc/ci-templates.
+
+Copy the [template for GitHub to Zenodo Sandbox publication](https://github.com/hermes-hmc/ci-templates/blob/main/TEMPLATE_hermes_github_to_zenodo.yml)
+into the `.github/workflows/` directory in your repository, and rename it as you like (e.g. `hermes_github_to_zenodo.yml`).
+
+Go through the file, and look for comments marked with `# ADAPT`.
+Adapt the file to your needs.
+If you need help with how GitHub Action workflows work in general,
+have a look at the [documentation on GitHub](https://docs.github.com/actions).
+
+Add the workflow file to version control and push it.
+
+```{warning}
+If you haven't adapted the workflow file, and push it to the branch `main`, you will create an automatic publication at this point.
+```
+
+```bash
+git add .github/workflows/hermes_github_to_zenodo.yml
+git commit -m "Configure automatic publication with HERMES"
+git push
+```
+
+## Automatic publication workflow
+
+```{admonition} Congratulations!
+You can now automatically publish your repository to Zenodo Sandbox!
+```
+
+````{margin}
+```{mermaid}
+---
+---
+flowchart TD
+    t(("Trigger\n (i.e. push)")) -->
+    ci1("Run Harvest, Process,\n Curate steps") -->
+    pr1("Create Curation PR") -->
+    d{"Accept?"}
+    d -->|Merge| ci2("Run Deposit &\n Postprocess step") --> pr2("Create PR from\n Postprocessing") --> e((("End")))
+    d -->|Close| ci3("Cleanup") --> e((("End")))
+```
+````
+
+Now, whenever the GitHub Actions workflow is triggered, it will publish a new version of your repository to Zenodo Sandbox.
+If you haven't adapted the workflow file, this will happen whenever you push to your `main` branch.
+
+The diagram on the right shows the different steps that will happen each time.
+
+When the workflow runs, it harvests and processes the metadata from Git and your `CITATION.cff` file,
+and creates a new pull request in your repository.
+You then have the chance to curate the metadata, i.e., make sure that it looks the way you want.
+If you merge the pull request, the actual publication is created, 
+and the HERMES configuration file is updated 
+with the ID of the publication.
+This is needed so that future published versions are collected under the same [*concept DOI*](https://help.zenodo.org/#versioning).
+
+If the updates from the publication actually change the configuration file, or the `CITATION.cff` file,
+HERMES will create another pull request after publication.
```

Comment:
This reads rather cryptic... it's not the *updates from the publication* but the postprocessing that changes stuff...

Reply:
Okay, will adapt.